Code/Framework/AzToolsFramework/Tests/Prefab/Overrides/PrefabInspectorOverrideTestFixture.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.hxx>
+#include <AzToolsFramework/UI/PropertyEditor/ComponentEditor.hxx>
+#include <Prefab/Overrides/PrefabInspectorOverrideTestFixture.h>
+
+namespace UnitTest
+{
+    void PrefabInspectorOverrideTestFixture::SetUpEditorFixtureImpl()
+    {
+        // Enable feature flags for DPE and InspectorOverrideManagement
+        AZ::SettingsRegistryInterface* registry = AZ::SettingsRegistry::Get();
+        registry->Set("/O3DE/Preferences/Prefabs/EnableInspectorOverrideManagement", true);
+        registry->Set("/O3DE/Autoexec/ConsoleCommands/ed_enableDPE", true);
```

Comment:
Can `EnableInspectorOverrideManagement` be imported from PrefabEditorPreferences if we make it public?

Reply:
Yes, I wanted to avoid putting the static string in the header file as that'll make a copy in every cpp that includes it. To avoid this, I made it an extern and put it in the header now.